src/pages/en/reports/accessibility/2022-07.md

```diff
@@ -86,7 +86,9 @@ Issue present in 85.7% of emails tested.
 
 ##### Why it’s needed:
 
-According to the [2021 WebAIM screen reader user survey](https://webaim.org/projects/screenreadersurvey9/#finding) 67.7% use headings as their primary way of finding information. Having an `<h1>` in the email means they can quickly and easily find the email in the page and the information inside it.
+According to the [2021 WebAIM screen reader user survey](https://webaim.org/projects/screenreadersurvey9/#finding) 67.7% use headings as their primary way of finding information. Having an `<h1>` in the email means they can quickly and easily find the email in the page and the information inside it. 
+
+However not all emails require headings, if it's a very simple email with no structure to the content (just a couple of paragraphs of text) then a heading may not be needed.
```

Comment:
Looks good. Just some grammar/punctuations changes:

> However, not all emails require headings. If it is a very simple email with no structure to the content (e.g. just a couple of paragraphs of text), then a heading may not be needed.

Assuming it is true that an email with no structured content does not require headings from an accessibility POV, the second part of this paragraph can be re-written to be less uncertain:

> For instance, a very simple email with no structure to the content (e.g. just a couple of paragraphs of text) does not require any headings.

Reply:
yeah good call, that sounds much better